env.in

```diff
@@ -54,6 +54,14 @@ else
   PATH="@abs_top_srcdir@/bin":$PATH
 fi
 
+# Tests are calibrated to be compiled (they're much slower when
+# interpreted) so make sure they can be auto-compiled upon 'make
+# check'.
+if test "$XDG_CACHE_HOME" = ""; then
+    XDG_CACHE_HOME="${TMPDIR:-/tmp}/guile-fibers-cache"
```

Comment:
This is terrible w.r.t. security.
If they are meant to be compiled, just compile them (with .scm->.go rules), or put them in a subdirectory of the build directory.

Reply:
Also, if you put them in a subdirectory of the build directory, you might as well do this unconditionally.

Reply:
Addressed in #113.